lodash.mergewith/index.js

```diff
@@ -1001,7 +1001,7 @@ function baseMergeDeep(object, source, key, srcIndex, mergeFunc, customizer, sta
       if (isArguments(objValue)) {
         newValue = toPlainObject(objValue);
       }
-      else if (!isObject(objValue) || (srcIndex && isFunction(objValue))) {
+      else if (!isObject(objValue) || isFunction(objValue)) {
```

Comment:
This is backport from new version https://github.com/snyk/lodash/blob/4.17.12-pre/lodash.js#L3668